assignments/lambda-classes.js

```diff
@@ -1 +1,138 @@
 // CODE here for your Lambda Classes
+class Person {
+    constructor(attrib) {
+        this.name = attrib.name;
+        this.age = attrib.age;
+        this.location = attrib.location;
+        this.gender = attrib.location;
+        }
+        speak() {
+            return `Hello my name is ${this.name}, I am from ${this.location}.`;
+        }
+    }
+
+
+class Instructor extends Person {
+    constructor(instructorAttrib) {
+    super(instructorAttrib);
+    this.specialty =   instructorAttrib.specialty;
+    this.favLanguage = instructorAttrib.favLanguage;
+    this.catchPhrase = instructorAttrib.catchPhrase;
+    }
+    demo(subject) {
+        return `Today we are learning ${subject}.`;
+    }
+    grade(student,subject){
+        return `${student.name} receives a perfect score in ${subject}.`
+    }
+}
+class Student extends Person {
+constructor(studentAttrib) {
+    super(studentAttrib);
+    this.previousBackground = studentAttrib.previousBackground;
+    this.className = studentAttrib.className;
+    this.favSubjects = studentAttrib.favSubjects;
+        }
+    listSubjects() {
+        return `${this.student}'s favorite subjects are ${this.favSubjects}.`;
```

Comment:
```suggestion
        return `${this.student}'s favorite subjects are ${this.favSubjects.toString()}.`;
```
As we're returning a string, I feel like it would be good to return the array as a string rather than an array. :)